rotkehlchen/chain/ethereum/modules/convex/decoder.py

```diff
@@ -42,6 +42,7 @@
 
 DEPOSIT_EVENT = b'\x9eq\xbc\x8e\xea\x02\xa69i\xf5\t\x81\x8f-\xaf\xb9%E2\x90C\x19\xf9\xdb\xday\xb6{\xd3J_='  # noqa: E501
 REWARD_ADDED = b'\xde\x88\xa9"\xe0\xd3\xb8\x8b$\xe9b>\xfe\xb4d\x91\x9ck\xf9\xf6hW\xa6^+\xfc\xf2\xce\x87\xa9C='  # noqa: E501
+CONVEX_POOLS = read_convex_data_from_cache()
```

Comment:
don't initialize it as a constant variable. If you check curve is an attribute because they can be reloaded as it is done here https://github.com/rotki/rotki/blob/1eebfdf6c9eba0ddfb4c4659cd66346cefdd9b0a/rotkehlchen/chain/ethereum/modules/curve/decoder.py#L156